### PR TITLE
Add support for terraform 0.13

### PR DESCRIPTION
**Pull request by mariux**
- Fix comlete test by removing support for broken aws provider versions
- Add support for terraform 0.13
- Prepare terraform 0.14 support
- Update build-tools to 0.6.1 to run tests with terraform 0.13.2

### CODE REVIEW

**Comment by mariux** [comment]
[ch1106]

File: test/unit-complete/main.tf [hunk 1]
```diff
@@ -10,9 +10,17 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+terraform {
+  required_providers {
+    # replica support was added in 2.58
+    # 3.5.0 to 3.6.0 is broken due to https://github.com/terraform-providers/terraform-provider-aws/issues/15115
+    aws = ">= 2.58, < 4.0, != 3.5.0, != 3.6.0"
```
Comment (soerenmartius) [inline test/unit-complete/main.tf:17]: Oh nice! Actually, I didn't even know that this is possible!

**Review by soerenmartius — APPROVED** [review]
lgtm!

File: test/unit-complete/main.tf (hunk 1 above)
Comment (mariux) [reply test/unit-complete/main.tf:17]: too sad it is needed...

**Pull request merged**